### DISCUSSION TITLE
Welcome to advene discussions

### DISCUSSION THREAD

**Discussion by oaubert** [oaubert/advene#63] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sportegioco** [comment]
Hello,
i have included in the advene guide:
https://github.com/oaubert/advene/wiki/AdveneUserGuide
the chapter on advene website export 
Please correct the English if there are any errors. 
Thank you

**Reply by oaubert** [reply]
Thanks a lot for this great contribution, I only made small adjustments.
